### PR TITLE
Add passing function to custom translation

### PR DESCRIPTION
**Pull request by adriankwiat**
I added this because in my language noun form depends on the number and simply changing the string will not help.

### CODE REVIEW

**Comment by adriankwiat** [comment]
Where it is implemented? I can't find it.

**Comment by VladimirMikulic** [comment]
Check out the "Custom Translations" section in the README.md to learn more about this newly added feature. Thanks.

**Comment by adriankwiat** [comment]
Yes, but this PR extends translation feature. Now you can pass function to `confirmImagesDeletes`.

**Comment by VladimirMikulic** [comment]
@adriankwiat messages are formatted in the way that it is no longer necessary to pass a function.

**Comment by adriankwiat** [comment]
In my language (Polish) i need to know number to choose correct noun form. For example: "2 obrazy", but "5 obrazów". There is no way to get it without a function.

**Comment by VladimirMikulic** [comment]
@adriankwiat could you rebase this branch?

**Comment by sashadev-sky** [comment]
sorry for the delay here, ill do a rebase test and merge if all checks out :) Thanks @adriankwiat @VladimirMikulic

File: README.md [hunk 1]
```diff
@@ -718,6 +718,21 @@ var translation = {
 };
 ```
 
+For `confirmImagesDeletes` you can pass a function that returs a string.
```
Comment (sashadev-sky) [inline README.md]:
```suggestion
For `confirmImagesDeletes` you can pass a function that returns a string.
```

File: README.md [hunk 2]
```diff
@@ -718,6 +718,21 @@ var translation = {
...
+This is useful for languages where noun form depends on the number:
```
Comment (sashadev-sky) [inline README.md]:
```suggestion
This is useful for languages where noun form depends on the number:

```

File: dist/leaflet.distortableimage.js [hunk 3]
```diff
@@ -55,7 +55,14 @@ L.DomUtil = L.extend(L.DomUtil, {
       return;
     }
 
-    var warningMsg = n + ' ' + this.translation.confirmImagesDeletes;
+    var translation = this.translation.confirmImagesDeletes;
+    var warningMsg = '';
+
+    if(typeof translation === 'function') {
```
Comment (sashadev-sky) [inline dist/leaflet.distortableimage.js]:
```suggestion
    if (typeof translation === 'function') {
```

**Comment by adriankwiat** [comment]
Thanks, I'll open a new PR.

**Pull request closed without merging**